client/src/pages/private/SiteParticipantsTable.js

```diff
@@ -38,9 +37,21 @@ export default ({ siteId }) => {
         method: 'GET',
       });
 
+      const testLookup = async (row) => {
+        const postalCode = row.participantJoin.body.postalCode;
+        const api_test = await fetch(`/api/v1/coords?postalCode=${postalCode}`, {
+          headers: { 'Authorization': `Bearer ${store.get('TOKEN')}` },
+          method: 'GET',
+        });
+        console.log(await api_test.json());
+      }
+
+
       if (response.ok) {
         const participants = await response.json();
         const rowsData = participants.map((row) => {
+          testLookup(row);
+
```

Comment:
I added this here to have an interactive demo within the spike. For each entry in the Site Participants table, it makes a query to get that participant's latlong.